src/pytypes/onefuzztypes/models.py

```diff
@@ -340,7 +340,7 @@ class AgentConfig(BaseModel):
 
 class TaskUnitConfig(BaseModel):
     instance_id: UUID
-    logs: str
+    logs: Optional[str]
```

Comment:
Could this cause issues on the agent side if the property isn't present?

Reply:
This field is optional on the client side already
https://github.com/microsoft/onefuzz/blob/457f03e4ae32c140c288327b2daf72db1c6b7b8b/src/agent/onefuzz-agent/src/tasks/config.rs#L49